src/borg/repository.py

```diff
@@ -1123,8 +1107,7 @@ def recover_segment(self, segment, filename):
                 data = data[size:]
 
     def read(self, segment, offset, id, read_data=True):
-        """
-        Read entry from *segment* at *offset* with *id*.
+        """Read entry from *segment* at *offset* with *id*.
 
         If read_data is False the size of the entry is returned instead and integrity checks are skipped.
```

Comment:
Ok, true -- we can clean them up / make them consistent sometime, separate change.